src/internal-context/load.ts

```diff
@@ -81,7 +81,8 @@ const createInternalContext = async (moduleOptions: ModuleOptions, nuxt = useNux
       const pageFiles = pagesContentPath.tryUse()
 
       if (pageFiles && pageFiles.length) {
-        rootProjectFiles.push(...pageFiles)
+        // replace filenames like [...path].vue with ?...path?.vue because [ and ] are reserved in glob matching
+        rootProjectFiles.push(...pageFiles.map(p => p.replaceAll(/\[(\.+)([^.].*)\]/g, '?$1$2?')))
```

Comment:
Should it be with ? or perhaps we escape using \[?

Reply:
Hi. Can you elaborate a little what do you mean with "with ?" ?

Reply:
Yes, forgive me - is it better to do `pages/to/\[...path\].vue` than `pages/to/?...path?.vue`?

Reply:
Well, that's the thing. I tried escaping them and all kinds of tricks. But only thing that seemed to work were the question marks. I think this might be an upstream problem in tailwindcss (or even the glob library). But, for now, this change will solve the problem with those paths.